### ISSUE TITLE
Alpha value 0.00 displays as fully opaque

### ISSUE THREAD

**Issue by jarlarntzen** [OpenShot/openshot-qt#19]
Unless I'm wrong, an alpha value of 1.00 is fully opaque and an alpha value of 0.00 is fully transparent. At least this applies to the alpha channel in PNGs.

After importing 8 pictures on the timeline and using the "Add to timeline" dialog (which works very well btw), I set the Fade to "Fade In & Out" and hit OK. 

This correctly created the slideshow with proper transitions, zooming and fading but when I selected a single image on the timeline and had a look at the Alpha in Properties it looked a bit odd. 
The image is fully visible when alpha is 0.00 and fully transparent when alpha is 1.00 while I would have thought it should be the exact opposite way.

Cheers,
Jarl

**Comment by jarlarntzen** [comment]
This happened no OSX El Capitan BTW. I havn't tested this on Ubuntu yet.

**Comment by jonoomph** [comment]
This is a great question / discussion. I've gone back and forth on this a couple times, primarily because I first named this property "transparency", since I was sticking closely to the ImageMagick API. And if something is 0% transparent, it's 100% visible. Later, I switched image backends (from ImageMagick to Qt), and the property switched from "transparency" to "opacity", but by this point I already had a lot of things coded. 

Anyway, I am open to renaming the property again, or flipping the range. Clearly "alpha" has a different meaning than "transparency". =)

**Comment by jarlarntzen** [comment]
I suppose naming it "Transparency" would be: 
1. More casual-user/beginner friendly (in the general spirit of OpenShot)
2. Consistent with the range as it is now.

**Comment by Bugsbane** [comment]
Almost everyone knows what transparency is. Many people do not know the meaning of opacity, or alpha. Either way, I'd display as a percentage, rather than a 0-1 range, as everyone understands what range a percentage covers, while that's not necessarily going to be obvious for a 0-1 range (unless you have a slider or something).

**Comment by jonoomph** [comment]
Yep, I agree. I've updated the label to "Transparency" for now... 

And my goal is to eventually have more rich editing options, such as sliders / spinners / etc... I just haven't had time to mess with that yet. It's pretty basic property editing at this point.

**Comment by jonoomph** [comment]
Okay, I've been doing some thinking about this one. I've fixed (i.e. flipped) the alpha values in libopenshot, so it works as most people will expect it to. This could potentially break some projects using our beta, so I've added code to upgrade those projects (when they run the new version), and it will automatically flip all their alpha values to the "new" correct values.

Also, renaming alpha for now, to keep everything consistent. Dang.. I just wished I caught this earlier... haha. Anyway, it should work correctly moving forward, as 'alpha'.  Thanks again.
